draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -802,6 +802,14 @@ informative:
         org: The Sage Developers
     target: https://www.sagemath.org
     date: 2019
+  W19:
+    title: An explicit, generic parameterization for the Shallue--van de Woestijne map
+    target: https://github.com/cfrg/draft-irtf-cfrg-hash-to-curve/doc/svdw_params.pdf
```

Comment:
Is there any chance you can put this on ePrint? (The current link doesn't work, so I can't review the rest of the PR!)

Reply:
I can, though there will be some delay before it's accepted.

The PDF is included in this PR, or you can find the same file at https://github.com/kwantam/draft-irtf-cfrg-hash-to-curve/raw/generalized_svdw/svdw_general.pdf

Reply:
Do you have a latex source for me to make a PR for that document?

Reply:
The source isn't public right now because it relies on some LaTeX sources that I haven't scrubbed for release. If the proposed edits are small, we can probably handle them in this thread. If you don't think that will work, I'll need to spend some time to make a version of the sources that I can make public.